crates/sui/src/validator_commands.rs

```diff
@@ -182,6 +184,18 @@ pub enum SuiValidatorCommand {
         #[clap(long)]
         validator_address: Option<SuiAddress>,
     },
+    /// Sui native bridge committee member registration
+    UpdateBridgeCommitteeNodeUrl {
+        #[clap(long)]
+        bridge_authority_url: String,
```

Comment:
can you add a comment here so it shows on the help for the command? It is self explanatory but still